translation.rst

```diff
@@ -329,6 +326,44 @@ The translator service is accessible in PHP templates through the
         array('%count%' => 10)
     ) ?>
 
+Extracting Translation Contents and Updating Catalogs Automatically
+-------------------------------------------------------------------
+
+The most time-consuming task when translating an application is to extract all
+the template contents to be translated and to keep all the translation files in
+sync. Symfony includes a command called ``translation:update`` that helps you in
+these tasks.
+
+The "safe mode" of this command uses the ``--dump-messages`` option to output
+in the command console the strings to be translated:
+
+.. code-block:: terminal
+
+    # shows the strings to be translated into French for app/Resources/ templates
+    $ ./app/console translation:update --dump-messages fr
+
+    # shows the strings to be translated into English for the AppBundle templates
+    $ ./app/console translation:update --dump-messages en AppBundle
+
+The "advanced mode" of this command uses the ``--force`` option to actually
+update the contents of the translation files to add the missing strings:
+
+.. code-block:: terminal
+
+    # updates the French translation file to add the missing strings
+    # found on app/Resources/ templates
+    $ ./app/console translation:update --force fr
+
+    # updates the English translation file to add the missing strings
+    # found on AppBundle templates
+    $ ./app/console translation:update --force en AppBundle
```

Comment:
I kind of think we should only show `--force` and mention in a comment (e.g. "without `--force`, the missing strings are only rendered to the screen")